reference/latest/src/main/java/com/example/docs/enchantment/ModEnchantmentEffects.java

```diff
@@ -0,0 +1,24 @@
+package com.example.docs.enchantment;
+
+import com.mojang.serialization.MapCodec;
+
+import net.minecraft.enchantment.effect.EnchantmentEntityEffect;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.Registry;
+import net.minecraft.util.Identifier;
+
+import com.example.docs.FabricDocsReference;
+import com.example.docs.enchantment.effect.LightningEnchantmentEffect;
+
+//#entrypoint
+public class ModEnchantmentEffects {
+	public static MapCodec<LightningEnchantmentEffect> LIGHTNING_EFFECT = register("lightning_effect", LightningEnchantmentEffect.CODEC);
+
+	private static <T extends EnchantmentEntityEffect> MapCodec<T> register(String id, MapCodec<T> codec) {
+		return Registry.register(Registries.ENCHANTMENT_ENTITY_EFFECT_TYPE, Identifier.of(FabricDocsReference.MOD_ID, id), codec);
+	}
+
+	public static void registerModEnchantmentEffects() {
+		System.out.println("Registering EnchantmentEffects for" + FabricDocsReference.MOD_ID);
```

Comment:
Don't use Sysout for logging stuff. Either use the provided LOGGER instance or don't print it.

_(technically the mod users or the modpack creators, who will use the mod later on, don't rly need to know when something got initialized anyway imho.)_

Reply:
> (technically the mod users or the modpack creators, who will use the mod later on, don't rly need to know when something got initialized anyway imho.)

This may be true, but this method is used to load the `ModEnchantmentEffects` class and Checkstyle prohibits having empty methods